src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -101,24 +74,6 @@ void addOrUpdateProfileToExistingCredentialsFile() throws IOException {
         assertEquals(expected, given);
     }
 
-    /*
-     * Test writing a new credentials profile to an existing credentials file with custom suffix.
-     */
-    @Test
-    void addOrUpdateProfileToExistingCredentialsFileCustomSuffix() throws IOException {
-        final StringReader credentialsReader = new StringReader(existingCredentials);
-        final StringWriter credentialsWriter = new StringWriter();
-        final Credentials credentials = new Credentials(credentialsReader, environmentWithCustomSuffix);
-
-        credentials.addOrUpdateProfile(roleName, accessKey, secretKey, sessionToken);
-        credentials.save(credentialsWriter);
-
-        String expected = existingCredentials + "\n\n" + manualRoleCustomSuffix;
-        String given = org.apache.commons.lang.StringUtils.remove(credentialsWriter.toString().trim(), '\r');
-
-        assertEquals(expected, given);
-    }
-
```

Comment:
:fire: suffix customization was a workaround for the workaround causing problems for people using this with downstream tools. Removing the writes to ~/.aws/config makes testing suffix customization unnecessary.